docs/products/flink/overview.rst

```diff
@@ -0,0 +1,97 @@
+
+Our product page provides you detailed information about the available plans and pricing. 
+
+
+Managed service features
+------------------------
+
+**Default deployment configuration**
+  Aiven for Apache Flink is configured to use the `HashMap state backend <https://ci.apache.org/projects/flink/flink-docs-stable/api/java/org/apache/flink/runtime/state/hashmap/HashMapStateBackend.html>`_. This means that the amount of `state <https://ci.apache.org/projects/flink/flink-docs-release-1.13/docs/concepts/stateful-stream-processing/#what-is-state>`_ kept depends on the amount of memory available in the cluster. 
+
+  The Flink cluster executes applications in `session mode <https://ci.apache.org/projects/flink/flink-docs-release-1.13/docs/deployment/overview/#session-mode>`_, so you can deploy multiple Flink jobs on the same cluster to use the available resources effectively.
+
+**Scaling a Flink cluster**
+  Each node is equipped with a TaskManager and JobManager. We recommend that you scale up your cluster to add more CPU and memory for the TaskManager before attempting to scale out. This approach makes the best use of the resources available in the nodes.
+
+  By default, each created TaskManager is configured with a single slot for best isolation between jobs. We highly recommend that you adjust this setting for your use case.
+
+  .. note::
+     Adjusting the task slots per TaskManager requires a cluster restart.	
+
+**Cluster restart strategy**
+  The cluster’s default restart strategy is configured to Failure Rate. This controls Flink’s restart behaviour in cases of failures during the execution of jobs. Administrators can overwrite this setting in the advanced configuration options for the service.
+
+  For more information on the available options, see the `Apache Flink documentation on fault tolerance <https://ci.apache.org/projects/flink/flink-docs-master/docs/deployment/config/#fault-tolerance>`_.
+
+**Disaster recovery**
+  Aiven has configured periodic checkpoints to be persisted externally in object storage. These allow Flink to recover state and positions in the data streams to provide failure-free execution.
+
+**End-to-end security**
+  All services run on a dedicated virtual machine with end-to-end encryption, and all nodes are firewall-protected.
+
+**Cluster logging, metrics, and alerting**
+  Administrators can configure log and metrics integrations to Aiven services so that you can monitor the health of your service.
+
+  These integrations allow you to
+
+  * push service logs to an index in Aiven for OpenSearch
+  * push service metrics to M3, InfluxDB, or PostgreSQL services on Aiven
+  * create custom OpenSearch or Grafana dashboards to monitor the service 
+
+  The platform also has alert policies to notify you via email when a key metric rises above or below a set threshold, such as low memory or high CPU consumption.
```

Comment:
Is there any documentation on this already available? Like, what should I expect?

Reply:
Sort of, so I added a link to that ;)